.gitmodules

```diff
@@ -22,8 +22,8 @@
 	branch = scorpio_classic
 [submodule "cosp2"]
 	path = components/eam/src/physics/cosp2/external
-	url = git@github.com:CFMIP/COSPv2.0.git
-	branch = CESM_v2.1.4
+	url = git@github.com:bartgol/COSPv2.0.git
+	branch = bartgol/fix-cosp_optical_inputs
```

Comment:
I have to change this. That branch got merged into the main cosp branch...

Reply:
Is this ready or still needs a change?